app.py

```diff
@@ -0,0 +1,87 @@
+
+import boto3
+import nexmo
+import random
+import string
+from flask import Flask, request
+app = Flask(__name__)
+app.config.from_pyfile('.env')
+sqs = boto3.client('sqs')
+
+
+@app.route('/')
+def index():
+    return "Success! Endpoints available: /add and /process."
+
+
+# store an SMS message to SQS FIFO for later sending
+@app.route('/add', methods=['POST'])
+def add():
+
+    if request.method == 'POST':
```

Comment:
Why do we need to bracket this code in this if statement, if the route method is defined as a POST already?

Reply:
I do it as good practice. If tests existed the added if statement would allow for more complete assertions. Short answer: not really needed in this code. Do you think I should remove it?